packages/babel-helper-module-transforms/src/rewrite-live-references.js

```diff
@@ -182,7 +182,8 @@ const rewriteReferencesVisitor = {
       ref.loc = path.node.loc;
 
       if (
-        path.parentPath.isCallExpression({ callee: path.node }) &&
+        (path.parentPath.isCallExpression({ callee: path.node }) ||
+          path.parentPath.isTaggedTemplateExpression({ tag: path.node })) &&
```

Comment:
Not related to this PR, but the following case also has been broken for the same reason:
```js
foo3?.()
```
Can you also fix the `OptionalCallExpression` case?

Reply:
done